src/actions/resize/crop.ts

```diff
@@ -1,28 +1,21 @@
 import Action from "../Action";
```

Comment:
I don't think there should be a crop action, crop_mode is a param of Resize

Reply:
At this point, I just refactored the existing crop to use Params,
However, I can see how in the very near future we'll join all Resize into a single class.

We'll do it on a separate PR as it'll change a few files.